### ISSUE TITLE
Add an attribute that indicates you only want to serialize the Id and Type property

### ISSUE THREAD

**Issue by adcloeshelle** [jsonyte/jsonyte#75]
The previous serializer we used has the ability to specify that you only want to serialize the Id and Type property of a specific resource. For example:

```
[JsonApiResource("other-model")]
public class OtherModel
{
    public String Id { get; }

    public string OtherValue { get; }
}

[JsonApiResource("model-with-attribute")]
public class ModelWithAttribute
{
    public string Id { get; set; }

    public string Value { get; set; }

    public int IntValue { get; set; }

    [SerializeAs(Type = RelationshipSerializationTypeEnum.IdOnly)]
    public OtherModel OtherModel { get; set; }
}
```

This indicates that the serialization of `OtherModel` only gets serialized with the Id and Type property even if the `OtherValue` property is filled.

Is this something you're open to adding to the repo? If so, I can try to create a PR for this. I just wanted to make sure you were on board with this concept before creating the PR.

**Comment by robertcoltheart** [comment]
Yeah makes sense, basically you're wanting to describe whether a relationship is output as a compound document via `included:[]` or whether it's just a type/id relationship. If I'm reading this right, this would have no bearing on deserialization, only on serialization.

Feel free to go ahead with a PR. I'll have some opinions about naming 😉 , but I think the API you've described is sound. There's also going to be some edge-cases with array relationships.
